cogs/induct.py

```diff
@@ -235,7 +234,7 @@ async def _perform_induction(self, ctx: TeXBotApplicationContext, induction_memb
 
         await induction_member.add_roles(
             guest_role,
-            reason=f"{ctx.user} used TeX Bot slash-command: \"/induct\"",
+            reason=f'{ctx.user} used TeX Bot slash-command: "/induct"',
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes